content/episode/003-agile.md

```diff
@@ -0,0 +1,31 @@
+---
+title: "Agile Development"
+episode: "3"
+Description: "Al and Andrew talk about agile development. Their opinions on whether it is useful, and if so...the best ways to do it."
+podcast: "pdd-3.mp3"
+podcast_bytes: "37049639"
+podcast_duration: "00:52:53"
+date: 2018-12-05T00:21:00+00:00
```

Comment:
I set this to this Wednesday, because we may as well, seeing as we are so far from last Wednesday. Hopefully I'll get better at editing quickly.